ui/src/components/system/EnvConfig.vue

```diff
@@ -52,13 +53,13 @@
             <q-card-actions align="right" class="text-primary">
               <q-btn
                 v-close-popup
-                :label="$t('general.cancel')"
                 v-bind="qBtnStyle"
+                :label="$t('general.cancel')"
               />
               <q-btn
                 :label="$t('general.Submit')"
                 v-bind="qBtnStyle"
-                @click="setEnv(newVarKey, newVarDialogOpen)"
+                @click="setEnv()"
```

Comment:
Oh interesting, I don't know why these params were being passed in the first place?